velox/docs/functions/presto/conversion.rst

```diff
@@ -448,8 +450,22 @@ consistent with other supported cases of cast.
 
 ::
 
+  SELECT cast('infinity' as real); -- Infinity
```

Comment:
presto:test> SELECT cast('infinity' as real);
Query 20231120_033856_00032_a9j2g failed: Cannot cast 'infinity' to REAL

Reply:
Same as https://github.com/facebookincubator/velox/pull/7629/files#r1408606756. Do I have anything missing?